app/repositories/Repository.py

```
@@ -7,7 +7,8 @@ def __init__(self, db: Session, model):
         self.model = model
 
     def get_all(self):
-        return self.db.query(self.model).all()
+        # NOTE TO @XpnsiveSharks: filtered out na yung mga soft deleted users. Or should I create a separate repository method instead of `get_all()`?
+        return self.db.query(self.model).filter(self.model.deleted_at == None).all()
```

Ruff (0.12.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)